packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -297,6 +344,16 @@ abstract class WidgetsBinding extends BindingBase with GestureBinding, RendererB
       observer.didChangeMetrics();
   }
 
+  /// Called when the platform text scale factor changes.
+  ///
+  /// See [Window.onTextScaleFactorChanged].
+  @override
```

Comment:
the docs should be inherited, you shouldn't need to doc it here again

Reply:
Yeah, I thought so, but was following handleMetricsChanged pattern.  Removed here, and on handleMetricsChanged, above.